src/tv2-constants/enums.ts

```diff
@@ -50,14 +50,16 @@ export const enum PartType {
 }
 
 export const enum SourceType {
-	Kam = 'Kam',
-	Server = 'Server',
+	KAM = 'KAM',
+	SERVER = 'SERVER',
 	VO = 'VO',
-	Teknik = 'Teknik',
-	Grafik = 'Grafik',
-	EVS = 'EVS',
-	REMOTE = 'Ekstern',
-	DEFAULT = 'Default'
+	TEKNIK = 'TEKNIK',
+	GRAFIK = 'GRAFIK',
+	REPLAY = 'REPLAY',
+	REMOTE = 'REMOTE',
```

Comment:
I think I am out of the loop here. Did we decide to call 'EVS' 'REMOTE' in the code?
(I'm all for it, just want to know if this is the way we are going now :) )